crates/wal-protocol/src/lib.rs

```diff
@@ -242,397 +249,3 @@ impl MatchKeyQuery for Envelope {
         self.record_keys().matches_key_query(query)
     }
 }
-
```

Comment:
This is currently not user (and will never be) a new draft for envelope redesign can be checked [here](https://github.com/restatedev/restate/pull/3854)